src/components/ContentFilters/ContentFilters.tsx

```diff
@@ -0,0 +1,77 @@
+import { useSelector, useDispatch } from 'react-redux';
+import Favorites from '../Favorites/Favorites';
+import Content from '../Content/Content';
+import useContentFilters from '../../hooks/useContentFilters';
+import { Dispatch, GlobalStateType } from '../../utils/types';
+import { requestQuantityNews, requestToggleOrientation } from '../../redux/actions';
+import toggleOrientation from '../../assets/toggleOrientation.svg';
+import {
+  SectionContainer,
+  ContentFiltersContainer,
+  ContentContainer,
+  Container,
+  Button,
+} from './styles';
+
+function ContentFilters() {
+  const globalState = useSelector((state: GlobalStateType) => state.news);
```

Comment:
Similar ao `NewHero`, aqui vale analisar se é mais interessante buscar cada um dos estados necessários (content, activeButton, etc) individualmente ou não.

Como pensar nisso? Existem muitos outros estados no global state? Qual a frequência de alteração desses estados. De qualquer forma, à princípio, [a style guide recomenda granularizar](https://redux.js.org/style-guide/#call-useselector-multiple-times-in-function-components) na maioria dos casos.